PIL/ImageOps.py

```diff
@@ -178,6 +178,27 @@ def crop(image, border=0):
         )
 
 
+def scale(image, factor, resample=Image.NEAREST):
+    """
+    Returns a rescaled image by a specific factor given in parameter.
+    A factor greater than 1 expands the image, between 0 and 1 contracts the
+    image.
+
+    :param factor: The expansion factor, as a float.
+    :param resample: An optional resampling filter. Same values possible as
+       in the PIL.Image.resize function.
+    :returns: An :py:class:`~PIL.Image.Image` object.
+    """
+    if factor == 1:
+        return Image.Image._new(image.im)
```

Comment:
`_new` is not a static method. You need to call it on the instance: `image._new()`.
Or you can use public method: `return image.copy()`.
Or you can remove this branch as @radarhere suggested.